dist/images/Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN INSTALL_PKGS=" \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
 	yum clean all && rm -rf /var/cache/yum/*
 
+RUN yum -y update
```

Comment:
you should reorder things; the previous command tries to clean up all of yum's caches to make the final image smaller

Reply:
My logic was that the update should run on whatever is installed.
If the update run before the clean, we will end up with updates on packages that will be cleaned later -> longer build.

Reply:
@shahar-klein instead of a separate layer through `RUN` command can we dot his

```
	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && yum -y update && yum clean all && rm -rf /var/cache/yum/*
```

Reply:
Good idea, I'll do that, thanks